src/test/java/enmasse/kafka/bridge/KafkaClusterTestBase.java

```diff
@@ -0,0 +1,48 @@
+package enmasse.kafka.bridge;
```

Comment:
Sure. tbh I wasn't sure whether to include the headers or not (since both projects are ASL2 and none of the other files in this project have headers), but having a heading in one file and not in the other is careless.